compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -399,18 +399,22 @@ internal class ComposeSceneMediator(
         }
     }
 
-    private fun subscribe(component: Component) {
-        component.addInputMethodListener(inputMethodListener)
-        component.addFocusListener(focusListener)
-        component.addKeyListener(keyListener)
-        component.subscribeToMouseEvents(mouseListener)
+    private fun subscribeToInputEvents() {
```

Comment:
just curiosity, what was wrong with previous shape here?

Reply:
You mean that it used to take the component to register with as a receiver?
It wasn't needed and was error prone, as the component has to be the same one in all cases.